packages/grafana-ui/src/components/Select/Select.tsx

```diff
@@ -50,6 +50,7 @@ export interface CommonProps<T> {
   onOpenMenu?: () => void;
   onCloseMenu?: () => void;
   tabSelectsValue?: boolean;
+  hideTextValue?: boolean;
```

Comment:
this property on Select does not look like it is needed, can't see that it is used anywhere.

Reply:
Right now it's being controlled by ExploreToolbar and passed down as a prop so it prevents hiding text from other Select components as they may use different breakpoints.

Reply:
but cannot see it being used anywhere, it's set on the SelectComponent (which does not have any such property)

Reply:
Ah, you're absolutely right. My bad. I missed to remove it from there as I forwarded this prop via value object.